datadog_checks_base/datadog_checks/base/checks/libs/prometheus.py

```diff
@@ -0,0 +1,85 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from prometheus_client.metrics_core import Metric
+from prometheus_client.parser import _parse_sample, _replace_help_escaping
+
+
+def text_fd_to_metric_families(fd):
```

Comment:
1. Could you add details as comment and PR desc why it's breaking things. Can be helpful later.

2. Could you add a permalink to the original code as comment ?

Reply:
Added comments.